test/ex_machina_test.exs

```diff
@@ -171,14 +131,15 @@ defmodule ExMachinaTest do
     user = MyApp.ExMachina.create(:user, admin: true)
 
     assert user == %{id: 3, name: "John Doe", admin: true}
-    assert_received {:created, %{name: "John Doe", admin: true}}
```

Comment:
We can probably remove this test and just use the one below since we aren't doing the repo definition here anymore